Source/Public/Build-Bicep.ps1

```diff
@@ -42,7 +42,9 @@ function Build-Bicep {
         [Parameter(ParameterSetName = 'AsHashtable')]
         [switch]$AsHashtable,
 
-        [switch]$IgnoreDiagnostics
+        [switch]$IgnoreDiagnostics,
```

Comment:
For consistency and ease of reading we should add valid parameter sets to these two parameters.

Reply:
Fixed this